docs/meshstack.index.md

```diff
@@ -3,7 +3,12 @@ id: meshstack.index
 title: Overview
 ---
 
-Welcome to the Meshstack!
+Welcome to the Meshstack Documentation!
 
-This document is targeted at cloud architects and SREs.
+Meshstack is the name of our technology that powers the Meshcloud solution. This document is targeted at cloud architects and SREs and introduces the components of Meshstack and their interaction.
 
+Meshstack is a modular software platform solution that connects organizational processes like project and access management with cloud technology. It provides a unique administration interface to the organization while replicating configuration into attached cloud platforms.
```

Comment:
d: Do you mean 'unified' instead of ' unique' ?

Reply:
yes :)